.hgignore

```diff
@@ -1,8 +0,0 @@
-syntax: glob
```

Comment:
Please don't remove mercurial files. The main development of `go2xunit` is done [on bitbucket](https://bitbucket.org/tebeka/go2xunit) and github is a mirror for users who prefer it.